### PR TITLE
Reacting to TestHost changes

### PR DESCRIPTION
**Pull request by kichalla**
@dougbu For reference, following are the dependencies of Dnx runtime
https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Dnx.Runtime/project.json#L25

### CODE REVIEW

**Comment by kichalla** [comment]
@dougbu related to PR https://github.com/aspnet/Testing/pull/154

File: test/test.xunit.runner.aspnet/project.json [hunk 1]
```diff
@@ -1,7 +1,17 @@
 {
     "frameworks": {
-        "dnx451": { },
-        "dnxcore50": { }
+        "dnx451": {
+            "frameworkAssemblies": {
+                "System.Collections": "4.0.0.0",
```
Comment (dougbu) [inline test/test.xunit.runner.aspnet/project.json]: Why repeat this? Should be provided through `xunit.runner.aspnet`

File: src/xunit.runner.aspnet/project.json [hunk 2]
```diff
@@ -13,11 +13,17 @@
     "frameworks": {
         "dnx451": {
             "frameworkAssemblies": {
-                "System.Threading": ""
+                "System.Collections": "4.0.0.0",
+                "System.Threading": "",
+                "System.Xml": "4.0.0.0",
+                "System.Xml.Linq": "4.0.0.0",
+                "System.Xml.XDocument": "4.0.0.0"
```
Comment (dougbu) [inline src/xunit.runner.aspnet/project.json]: This looks heavier than I would expect. Does the runner really need all of these? If you haven't already, I recommend removing each and trying `dnu build`. Builds don't take very long for this project :smiley_cat:

**Comment by dougbu** [comment]
:watch:

File: test/test.xunit.runner.aspnet/project.json (hunk 1 above)
Comment (kichalla) [inline test/test.xunit.runner.aspnet/project.json]: good catch

File: src/xunit.runner.aspnet/project.json [hunk 3]
```diff
@@ -13,11 +13,17 @@
     "frameworks": {
         "dnx451": {
             "frameworkAssemblies": {
-                "System.Threading": ""
+                "System.Collections": "4.0.0.0",
+                "System.Threading": "",
```
Comment (kichalla) [inline src/xunit.runner.aspnet/project.json]: Q: Which is the correct way to reference assembly here? the exact version or `""`?

File: src/xunit.runner.aspnet/project.json (hunk 2 above)
Comment (kichalla) [inline src/xunit.runner.aspnet/project.json]: Sure, as discussed offline, these were found individually by doing build.

File: src/xunit.runner.aspnet/project.json (hunk 3 above)
Comment (dougbu) [inline src/xunit.runner.aspnet/project.json]: `""` for `dnx451`-specific dependencies. At least that's what we do throughout the MVC repo.

**Comment by kichalla** [comment]
Updated. @dougbu

**Comment by dougbu** [comment]
:shipit:

**Comment by kichalla** [comment]
dcc7e5d7e7d3904a161b9938265a7982e45cacef

**Pull request closed without merging**